DEPLOY.rst

```diff
@@ -48,6 +48,116 @@ creating a load balancer, and creating an autoscaling group.
 You can skip most of the *Project Configuration* section because that's
 already been done for this project, but review it to be sure.
 
+Autoscaling policies
+~~~~~~~~~~~~~~~~~~~~
+
+This part isn't covered in the Fabulaws docs. These are example policies
+to start with for the autoscaling group.
+
+Notifications
++++++++++++++
+
+These alarms are configured on the load balancer,
+actually.
+
+Alarm: ccsr-sam-production-backend-connection-errors
```

Comment:
Could rename this to 'Backend Connection Errors' for consistency with the other alarm names.